gradle/libs.versions.toml

```diff
@@ -153,6 +156,7 @@ swagger-ui = { module = "org.webjars:swagger-ui", version.ref = "swagger-ui" }
 androidx-work-runtime = { module = "androidx.work:work-runtime", version.ref = "androidx-work" }
 androidx-work-runtime-ktx = { module = "androidx.work:work-runtime-ktx", version.ref = "androidx-work" }
 
+androidx-browser = { module = "androidx.browser:browser", version.ref = "custom-tab" }
```

Comment:
Please name this using the same pattern as we do for other androidx dependencies. eg. androidx-browser